backend/src/packages/files/file.model.ts

```diff
@@ -1,14 +1,37 @@
+import { Model, type RelationMappings } from 'objection';
+
 import {
   AbstractModel,
   DatabaseTableName,
 } from '~/libs/packages/database/database.js';
+import { composeDatabaseRelationPath } from '~/libs/packages/database/libs/helpers/helpers.js';
+import { UserDetailsModel } from '~/packages/users/user-details.model.js';
 
 class FileModel extends AbstractModel {
   public 'url': string;
 
   public static override get tableName(): string {
     return DatabaseTableName.FILES;
   }
+
+  public static get relationMappings(): RelationMappings {
```

Comment:
Not sure if this relation for files is necessary

Reply:
Yes, you're right.
- [a7d9df5](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/118/commits/a7d9df5f326a071b27cb076c3d01f25cd7e26317)